flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -506,7 +491,7 @@ public void testGetKeysAndNamespaces() throws Exception {
         }
     }
 
-    @Test
+    @TestTemplate
     @SuppressWarnings("unchecked")
     public void testBackendUsesRegisteredKryoDefaultSerializer() throws Exception {
```

Comment:
```suggestion
    void testBackendUsesRegisteredKryoDefaultSerializer() throws Exception {
```
could be package private